code/arg_parser.py

```diff
@@ -0,0 +1,65 @@
+import os
+import yaml
+import logging
+import logging.config
+import argparse
+
+from lisibilite.Lisibilite import Lisibilite
+from config.AppConfiguration import LOG_DIR, LOG_CONFIG_FILE
+
+
+def init():
+    """
+    Method to init the application
+    :return: None
+    """
+    configureLogging()
+    logging.debug(f'{__name__} init')
+    logging.debug('Log configuration completed')
+    process()
+
+
+def configureLogging():
+    """
+    Method to configure the logging
+    :return: None
+    """
+    try:
+        logging.info('Setting up loggers')
+        if not os.path.exists(LOG_DIR):
+            os.makedirs(LOG_DIR)
+        logging.config.dictConfig(yaml.load(open(LOG_CONFIG_FILE), Loader=yaml.FullLoader))
+        logging.debug('Loggers initialized')
+    except Exception as e:
+        logging.error(f"Error in the Log Configuration. Error = {e}")
+        exit(1)
```

Comment:
Why do we need init and configure logging here. It is already present in the main right?

Reply:
i copied over the code in main, thought this would become the new main file. Do you want this to only parse the arguments and return?

Reply:
Yes, It would be good to keep this only for arg parse and as a class.